specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -9238,9 +9239,14 @@
           "description": "Value for Constant/Field Type: string.",
           "type": "string"
         },
-        "operator": {
-          "description": "Expression operator value Type: string.",
-          "type": "string"
+        "operators": {
+          "description": "Expression operator value Type: list of strings.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "x-ms-format": "dfe-string",
+            "description": "Type: string."
```

Comment:
Are operators really freeform strings.... or is it more like an enumeration type?
You'll be validating these on the server side I assume?

Reply:
Operators are freeform string since ExpressionV2 is generic and we don't want to make it ServiceNowV2 connector specific. This will be checked in backend.